setup.py

```diff
@@ -4,14 +4,17 @@
 with open('requirements.txt') as f:
     requirements = f.read().split()
 
+git_requirements = [r for r in requirements if r.startswith('git+')]
+requirements = [r for r in requirements if not r.startswith('git+')]
+print("User must install the following packages manually:\n" +
+        "\n".join(f' {r}' for r in git_requirements))
```

Comment:
@teddyrendahl thoughts on skipping this print if git_requirements is empty? It makes tons of spammy prints in all the conda builds (like, this file must get run 6+ times)

Reply:
Fine by me.